### PR TITLE
[JIT-1369] Fix Geyser build

### PR DESCRIPTION
**Pull request by esemeniuc**
Reported in discord: https://discord.com/channels/938287290806042626/1035260077629788172/1107407478909575271

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,5 @@
-FROM rust:1.64.0
+FROM rust:1.60.0
```
Comment (buffalu) [inline Dockerfile:1]: we should stay in sync with rust-version.sh by running the ./cargo below instead of normal cargo

Comment (esemeniuc) [reply Dockerfile:1]: this will be overridden during the build process. mainly changing this to prevent downloading rust again

Comment (esemeniuc) [reply Dockerfile:1]: its already using `./cargo`: https://github.com/jito-foundation/geyser-grpc-plugin/blob/91f2ce54c3d29ae8c73265b87b96145b30082230/Dockerfile#L45

Comment (buffalu) [reply Dockerfile:1]: yeah i see, mainly just point on the comment keeping it up to sync when its actually the ./cargo keeping it in sync

Comment (esemeniuc) [reply Dockerfile:1]: updated comment

**Review by buffalu — APPROVED** [review]

**Pull request merged**